### DISCUSSION TITLE
Is there any clear roadmap or direction of the project?

### DISCUSSION THREAD

**Discussion by MaxMorais** [undb-xyz/undb#1180] [Q&A]
Hi Guys! 

I found undb on my researches on github on the past days, and I'm astonished, cuz this project fits all of my desires, LOL.

Now, talking a little serious, is there a clear roadmap or direction the project will take on the next months?

Why I'm asking?

I was searching for an self-hosted alternative to Notion, when I found undb.
I know the project advertise that it's an `airtable` alternative, but from my point of view, it's much more closer to be a notion alternative also.

My plan, is to work on an integration with [Editor.js](https://editorjs.io/) on the next days, and create the widgets to embed the tables, and views on the documents.

So, my question is if the project is interested in this kind of feature?

**Comment by nichenqin** [comment]
Hi Max, thanks for you post. We have a roadmap project here https://github.com/orgs/undb-xyz/projects/1/views/2?query=is%3Aopen+sort%3Aupdated-desc
Feel free to add some feature if you need it : )

**Reply by draylegend** [reply]
link's broken. Can you please update it?

**Reply by raphaklaus** [reply]
Looks like this is the correct URL: https://github.com/orgs/undb-io/projects/1/views/2?query=is%3Aopen+sort%3Aupdated-desc

**Reply by raphaklaus** [reply]
@nichenqin Do you need help developing some of the items? It seems a great project and it doesn't deserve to be abandoned.
